common/src/KokkosFFT_Extents.hpp

```diff
@@ -33,7 +33,7 @@ namespace Impl {
 template <typename InViewType, typename OutViewType, std::size_t DIM = 1>
 auto get_extents(const InViewType& in, const OutViewType& out,
                  axis_type<DIM> axes, shape_type<DIM> shape = {},
-                 bool is_inplace = false) {
+                 [[maybe_unused]] bool is_inplace = false) {
```

Comment:
Why does it become needed only in this PR ?

Reply:
Actually, we should have encountered a compiler error due to the unused variable for C2C transform. 
For some reason, we did not get compiler errors before introducing the inplace C2C transform with the same `View`.

Reply:
Unexpected indeed, but I agree that it should be added anyway.